.harness/python3_sklearn_params_duplication.yaml

```diff
@@ -0,0 +1,39 @@
+inputSet:
```

Comment:
This input set is used in `zohar_` duplication pipeline and can be removed

Reply:
So, are you going to remove it in your PR?

Reply:
Will remove it